core/src/components/menu/test/basic/e2e.ts

```diff
@@ -28,13 +28,42 @@ test('menu: focus trap', async () => {
   await menu.waitForVisible();
 
   let activeElID = await getActiveElementID(page);
+  expect(activeElID).toEqual('start-menu');
+
+  await page.keyboard.press('Tab');
+  activeElID = await getActiveElementID(page);
   expect(activeElID).toEqual('start-menu-button');
 
+  // do it again to make sure focus stays inside menu
   await page.keyboard.press('Tab');
   activeElID = await getActiveElementID(page);
   expect(activeElID).toEqual('start-menu-button');
 });
 
+test('menu: preserve scroll position', async () => {
```

Comment:
I know mandatory Playwright tests do not start until next sprint, but could you write this in Playwright if you have time? Focus management has been tricky across browsers, so it would be great to test this in Firefox, Chrome, and Safari. 

If you don't have time this sprint I can make a ticket for next sprint. Let me know what you would prefer.

Reply:
I would go ahead and make the ticket; if I have time I can knock it out, but I'm not confident 🤔